opentelemetry-api/src/opentelemetry/_metrics/__init__.py

```diff
@@ -30,12 +30,12 @@
 
 from opentelemetry._metrics.instrument import (
     Counter,
-    DefaultCounter,
-    DefaultHistogram,
-    DefaultObservableCounter,
-    DefaultObservableGauge,
-    DefaultObservableUpDownCounter,
-    DefaultUpDownCounter,
+    NoOpCounter,
```

Comment:
Please update the corresponding test cases as well.

Reply:
Yeah I don't know what happened.... I thought i'd updated all the instances throughout the code but i forgot to save the changes to the test files 🤷 

Fixed now